code/modules/mob/living/carbon/combat.dm

```diff
@@ -208,6 +208,9 @@
 	tForce += get_strength()*10
 	tForce += offenseMutTackle()
 	tForce += bonusTackleForce()
+	var/is_real_champion = is_wearing_item(/obj/item/weapon/storage/belt/champion) && is_wearing_item(/obj/item/clothing/mask/luchador)
```

Comment:
if you're going to paste it 3 places, you may as well just make it a macro

Reply:
Where would I put the macro though?

Reply:
How about macros.dm?